stl/inc/cmath

```diff
@@ -1323,6 +1320,15 @@ _NODISCARD constexpr inline long double lerp(
     const long double _ArgA, const long double _ArgB, const long double _ArgT) noexcept {
     return _Common_lerp(_ArgA, _ArgB, _ArgT);
 }
+
+template <class _Ty>
+concept _Arithmetic = is_arithmetic_v<_Ty>;
+
+template <_Arithmetic _Ty1, _Arithmetic _Ty2, _Arithmetic _Ty3>

```

Comment:
This is guarded by `_HAS_CXX20`, but not `__cpp_lib_concepts`. Therefore, we should use SFINAE like `hypot()` above.